core/src/main/resources/io/cucumber/core/version.properties

```
@@ -1 +1,2 @@
-cucumber-jvm.version=${parent.version}
\ No newline at end of file
+cucumber-jvm.version=${parent.version}
+gherkin.version=${gherkin-messages.version}
```

Currently (RC3) we support Gherkin 5 by default and Gherkin 8+ (rule keyword) as an opt in module. However only JUnit 5 can accurately represent these outputs so I wouldn't enable it by default. Once we've finished the protobuf/ndjson output we'll probably enable Gherkin 8+ by default. 

Which is a really round about way to say that I think this should be: 
```suggestion
gherkin.version=${gherkin-vintage.version}
```